contracts/dtoken/src/common.rs

```diff
@@ -22,7 +22,7 @@ impl Contract {
 
     pub fn get_exchange_rate(&self, underlying_balance: WBalance) -> Ratio {
         if self.token.total_supply == 0 {
-            return self.initial_exchange_rate;
+            return self.initial_exchange_rate * RATIO_DECIMALS;
```

Comment:
@pahomas initial exchange rate should already be in Ratio representation (if not probably we would need to update initialization)